samples/DevTestLabs/SimpleUI/ClientApp/README.md

```diff
@@ -0,0 +1,9 @@
+This project was bootstrapped with [Create React App](https://github.com/facebookincubator/create-react-app).
```

Comment:
Probably fine to have 1 readme, can move these things to the top level readme on deployment